.travis.yml

```diff
@@ -1,15 +1,20 @@
 language: python
 sudo: false
-python:
-    - 2.6
-    - 2.7
-    - 3.3
-    - 3.4
-    - pypy
-    - pypy3
+
+env:
+  - TOXENV=py26
+  - TOXENV=py27
+  - TOXENV=py32
+  - TOXENV=py33
+  - TOXENV=py34
+  - TOXENV=pypy
+  - TOXENV=pypy3
+
 install:
-    - pip install .
+  - travis_retry pip install tox
+
 script:
-    - python setup.py test -q
+  - travis_retry tox
```

Comment:
Hmm, good question. I just copied the `.travis.yml` from whatever project happened to be nearest---in this case, [zope.i18n](https://github.com/zopefoundation/zope.i18n/blob/master/.travis.yml#L15). I'm not aware of any flaky tests there either. Maybe the original author @tseaver can provide some insight?